src/archiver.js

```diff
@@ -1,106 +1,150 @@
-                } else {
-                    console.log(`Did not find location for ${state.name} -> ${district.name} -> ${cvc.name}`)
+          const cvcs = await district.getCVCs(today);
+          console.log(
+            `Found ${cvcs.length} current vaccination sites in ${state.name} -> ${district.name}`
+          );
+          const augmentedCVCs = await pMap(
+            cvcs,
+            async (cvc) => {
+              const rawData = cvc.rawData;
+              if (locations[cvc.name.toLowerCase()]) {
+                rawData.address = locations[cvc.name.toLowerCase()];
+                console.log(
+                  `Found local location for ${state.name} -> ${district.name} -> ${cvc.name}`
+                );
+              } else {
+                console.log(
+                  `Did not find location for ${state.name} -> ${district.name} -> ${cvc.name}`
+                );
+              }
+              return rawData;
+            },
+            { concurency: 4 }
+          );
+          vaccinationsData.states[state.name].districts[district.name] = {
+            id: district.id,
+            name: district.name,
+            cvcs: augmentedCVCs,
+          };
+        },
+        { concurrency: 4 }
+      );
+    },
+    { concurrency: CONCURRENT_STATES }
+  );
 
-                }
-                return rawData
-            }, { concurency: 4 })
-            vaccinationsData.states[state.name].districts[district.name] = {
-                id: district.id,
-                name: district.name,
-                cvcs: augmentedCVCs
-            }
-        }, { concurrency: 4 })
-    }, { concurrency: CONCURRENT_STATES })
-
-    return {
-        slots: slotsData,
-        vaccinations: vaccinationsData
-    };
+  return {
+    slots: slotsData,
+    vaccinations: vaccinationsData,
+  };
 }
 async function main() {
-    const isroLocations = await getIsroCSVLocations(`${__dirname}/../data/isro.csv`)
-    const dump = await getFullDump(isroLocations);
-    for (const key of Object.keys(dump)) {
-        const filename = path.join('raw', 'v1', key, DateTime.now().setZone('Asia/Kolkata').toISO() + '.json.gz');
-        console.log(`Writing ${filename}`)
-        // Publish to s3
-        await publishS3('indiavaccine-cvc', filename, zlib.gzipSync(JSON.stringify(dump[key])));
-        // Publish to Git
-        fs.mkdirSync(path.dirname(filename), { recursive: true });
-        fs.writeFileSync(filename, zlib.gzipSync(JSON.stringify(dump[key])));
-        console.log(`Written ${filename}`)
-
+  const isroLocations = await getIsroCSVLocations(
+    `${__dirname}/../data/isro.csv`
+  );
+  const dump = await getFullDump(isroLocations);
+  for (const key of Object.keys(dump)) {
+    const filename = path.join(
+      "raw",
+      "v1",
+      key,
+      DateTime.now().setZone("Asia/Kolkata").toISO() + ".json.gz"
+    );
+    if (key === "vaccinationsData") {
+      const mappedFilename = path.join(
+        "raw",
```

Comment:
can you put this in something like processed/v1/ so it is different from where the unprocessed files are?